<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1024</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QStackedWidget" name="stackedWidget">
      <property name="currentIndex">
       <number>0</number>
      </property>
      <widget class="QWidget" name="page">
       <widget class="Line" name="line">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>80</y>
          <width>1231</width>
          <height>21</height>
         </rect>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Sunken</enum>
        </property>
        <property name="lineWidth">
         <number>20</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
       <widget class="QLabel" name="time_label">
        <property name="geometry">
         <rect>
          <x>110</x>
          <y>20</y>
          <width>261</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>00 : 00 : 00</string>
        </property>
       </widget>
       <widget class="QWidget" name="verticalLayoutWidget">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>100</y>
          <width>1013</width>
          <height>451</height>
         </rect>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QPushButton" name="app_button">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/application.png</normaloff>:/Images/application.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>true</bool>
             </property>
             <property name="default">
              <bool>true</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="calendar_button">
             <property name="minimumSize">
              <size>
               <width>250</width>
               <height>250</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">image: url(:/images/Images/schedule.png);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/schedule.png</normaloff>:/Images/schedule.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="autoDefault">
              <bool>false</bool>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="settings_button">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/gear.png</normaloff>:/Images/gear.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="Info_button">
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="Resources.qrc">
               <normaloff>:/Images/info.png</normaloff>:/Images/info.png</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>200</width>
               <height>200</height>
              </size>
             </property>
             <property name="flat">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="font">
              <font>
               <family>Ubuntu Thin</family>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Application</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_7">
             <property name="font">
              <font>
               <family>Ubuntu Thin</family>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Calendar</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_6">
             <property name="font">
              <font>
               <family>Ubuntu Thin</family>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Settings</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label">
             <property name="font">
              <font>
               <family>Ubuntu Thin</family>
               <pointsize>30</pointsize>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true"/>
             </property>
             <property name="text">
              <string>Info</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
       <widget class="QLabel" name="temperature_label">
        <property name="geometry">
         <rect>
          <x>590</x>
          <y>20</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="temperature_label_2">
        <property name="geometry">
         <rect>
          <x>650</x>
          <y>20</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Â°C</string>
        </property>
       </widget>
       <widget class="QLabel" name="temperature_label_3">
        <property name="geometry">
         <rect>
          <x>860</x>
          <y>20</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Ubuntu</family>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string>%</string>
        </property>
       </widget>
       <widget class="QLabel" name="humidity_label">
        <property name="geometry">
         <rect>
          <x>800</x>
          <y>20</y>
          <width>61</width>
          <height>41</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>30</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QPushButton" name="tempImg">
        <property name="geometry">
         <rect>
          <x>710</x>
          <y>20</y>
          <width>50</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="humidityImg">
        <property name="geometry">
         <rect>
          <x>900</x>
          <y>20</y>
          <width>50</width>
          <height>50</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">background:transparent;
border:0px;</string>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="icon">
         <iconset resource="Resources.qrc">
          <normaloff>:/Images/humidity.png</normaloff>:/Images/humidity.png</iconset>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QPushButton" name="alarmButton">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>20</x>
          <y>12</y>
          <width>80</width>
          <height>61</height>
         </rect>
        </property>
        <property name="styleSheet">
         <string notr="true">image: url(:/Images/alarm.png);</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_2">
       <widget class="QLabel" name="label_2">
        <property name="geometry">
         <rect>
          <x>440</x>
          <y>70</y>
          <width>241</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>APPLICATION</string>
        </property>
       </widget>
       <widget class="QPushButton" name="back_button">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_3">
       <widget class="QLabel" name="label_3">
        <property name="geometry">
         <rect>
          <x>450</x>
          <y>60</y>
          <width>151</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>SETTINGS</string>
        </property>
       </widget>
       <widget class="QPushButton" name="back_button_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>2000</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_4">
       <widget class="QLabel" name="label_4">
        <property name="geometry">
         <rect>
          <x>640</x>
          <y>100</y>
          <width>63</width>
          <height>20</height>
         </rect>
        </property>
        <property name="text">
         <string>INFO</string>
        </property>
       </widget>
       <widget class="QPushButton" name="back_button_3">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>100</width>
          <height>2000</height>
         </size>
        </property>
       </widget>
      </widget>
      <widget class="QWidget" name="page_5">
       <widget class="QPushButton" name="back_button_4">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>10</y>
          <width>30</width>
          <height>30</height>
         </rect>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="iconSize">
         <size>
          <width>50</width>
          <height>50</height>
         </size>
        </property>
       </widget>
       <widget class="QCalendarWidget" name="calendarWidget">
        <property name="enabled">
         <bool>true</bool>
        </property>
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>50</y>
          <width>1001</width>
          <height>481</height>
         </rect>
        </property>
        <property name="autoFillBackground">
         <bool>true</bool>
        </property>
        <property name="gridVisible">
         <bool>true</bool>
        </property>
        <property name="horizontalHeaderFormat">
         <enum>QCalendarWidget::SingleLetterDayNames</enum>
        </property>
        <property name="verticalHeaderFormat">
         <enum>QCalendarWidget::NoVerticalHeader</enum>
        </property>
        <property name="navigationBarVisible">
         <bool>true</bool>
        </property>
        <property name="dateEditEnabled">
         <bool>true</bool>
        </property>
       </widget>
       <zorder>calendarWidget</zorder>
       <zorder>back_button_4</zorder>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1024</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources>
  <include location="Resources.qrc"/>
 </resources>
 <connections/>
</ui>
